### PR TITLE
Update OpenShift template for registry viewer

### PR DESCRIPTION
**Pull request by johnmcollier**
**DO NOT MERGE UNTIL https://github.com/devfile/registry-support/pull/73 IS MERGED**

- Adds the viewer host config to the registry configmap
- Updates the resources for the index server container accordingly (higher resources needed to accommodate the viewer)
- Adds a startup probe to trigger the regeneration of the viewer

### CODE REVIEW

File: .ci/deploy/devfile-registry.yaml [hunk 1]
```diff
@@ -148,13 +171,20 @@ objects:
...
+      {
+        "Community": {
+          "url": "http://localhost:8080"
+        }
+      }
+
 parameters:
 - name: DEVFILE_INDEX_IMAGE
-  value: quay.io/devfile/devfile-index
+  value: quay.io/jcollier/devfile-index
```
Comment (elsony) [inline .ci/deploy/devfile-registry.yaml:183]: Is there any reason why we use the personal image instead of the one under devfile?

Comment (johnmcollier) [reply .ci/deploy/devfile-registry.yaml:183]: No, that was just there for testing and accidentally got left in

Comment (johnmcollier) [reply .ci/deploy/devfile-registry.yaml]: Fixed, thanks for catching that!

**Review by elsony — APPROVED** [review]
/lgtm

**Review by GeekArthur — APPROVED** [review]

**Comment by johnmcollier** [comment]
Tests overall fine, gonna merge. Python failure unrelated to these changes (flaky CI).

**Pull request merged**